test/foundry/utils/PermitSigUtils.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.13;
+
+contract PermitSigUtils {
+  bytes32 internal DOMAIN_SEPARATOR;
+
+  constructor(bytes32 _DOMAIN_SEPARATOR) {
+    DOMAIN_SEPARATOR = _DOMAIN_SEPARATOR;
+  }
+
+  // keccak256("Permit(address owner,address spender,uint256 value,uint256 nonce,uint256 deadline)");
```

Comment:
Does this comment refer to the byte representation of the following line?

Reply:
Yes, that is correct